app/src/androidTestKiwix/java/org/kiwix/kiwixmobile/tests/DownloadTest.java

```diff
@@ -103,8 +104,8 @@ public void downloadTest() {
         allOf(withText("Device"), isDisplayed()));
     appCompatTextView3.perform(click());
 
-    onView(withId(R.id.menu_rescan_fs))
-        .perform(click());
+    onView(withId(R.id.swiperefresh))
```

Comment:
I am failing download and network tests with 

`android.support.test.espresso.AmbiguousViewMatcherException: 'with id: org.kiwix.kiwixmobile:id/swiperefresh' matches multiple views in the hierarchy.
Problem views are marked with '****MATCHES****' below.`

Reply:
Hmm, so how is it supposed to work then? Isn't a swipeDown doing the action?

Reply:
You have given 2 layouts the same ID. The test doesn't know which one to swipe on.

Reply:
Aaa, now I get it, fix is coming.